.travis.yml

```diff
@@ -176,3 +177,18 @@ jobs:
       env:
         - MPLBACKEND=ps
         - PYTHON_VERSION=3.7.2
+
+    # Randomized testing and benchmarks
+    - stage: randomized
+      name: Randomized tests
+      <<: *stage_linux
+      language: python
```

Comment:
You can drop this it's defined already on https://github.com/Qiskit/qiskit-terra/pull/2620/files#diff-354f30a63fb0907d4ad57269548329e3R51